source/operations/install-deploy-manage/deploy-operator-kustomize.rst

```diff
@@ -0,0 +1,218 @@
+                      "port": 9443,
+                      "protocol": "TCP",
+                      "targetPort": 9443,
+                      "nodePort": 30433
+                  }
+              ],
+              "type": "NodePort"
+          }
+      }'
+
+
+#. Verify the Operator installation
+
+   Check the contents of the specified namespace (``minio-operator``) to ensure all pods and services have started successfully.
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl get all -n minio-operator
+
+   The response should resemble the following:
+
+   .. code-block:: shell
+
+      NAME                                  READY   STATUS    RESTARTS   AGE
+      pod/console-68d955874d-vxlzm          1/1     Running   0          25h
+      pod/minio-operator-699f797b8b-th5bk   1/1     Running   0          25h
+      pod/minio-operator-699f797b8b-nkrn9   1/1     Running   0          25h
+
+      NAME               TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
+      service/console    ClusterIP   10.43.195.224   <none>        9090/TCP,9443/TCP   25h
+      service/operator   ClusterIP   10.43.44.204    <none>        4221/TCP            25h
+      service/sts        ClusterIP   10.43.70.4      <none>        4223/TCP            25h
+
+      NAME                             READY   UP-TO-DATE   AVAILABLE   AGE
+      deployment.apps/console          1/1     1            1           25h
+      deployment.apps/minio-operator   2/2     2            2           25h
+
+      NAME                                        DESIRED   CURRENT   READY   AGE
+      replicaset.apps/console-68d955874d          1         1         1       25h
+      replicaset.apps/minio-operator-699f797b8b   2         2         2       25h
+
+
+#. Retrieve the Operator Console JWT for login
+
+   .. code-block:: shell
+      :class: copyable
+
+      kubectl apply -f - <<EOF
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: console-sa-secret
+        namespace: minio-operator
+        annotations:
+          kubernetes.io/service-account.name: console-sa
+      type: kubernetes.io/service-account-token
+      EOF
+      SA_TOKEN=$(kubectl -n minio-operator  get secret console-sa-secret -o jsonpath="{.data.token}" | base64 --decode)
+      echo $SA_TOKEN
+
+   
+#. Log into the MinIO Operator Console
+
+   If you configured the ``svc/console`` service for access through ingress, a cluster load balancer, you can access the Console using the configured hostname and port.
+
+   If you configured the service for access through NodePorts, specify the hostname of any worker node in the cluster with that port as ``HOSTNAME:NODEPORT`` to access the Console.
+
+   For example, a deployment configured with a NodePort of 30090 and the following ``INTERNAL-IP`` IP addresses can be accessed at ``http://172.18.0.2:30090``.
+   
+   .. code-block:: shell
+      :class: copyable
+
+      $ kubectl get nodes -o wide
```

Comment:
maybe there's a better way to find the IP addresses, this command is pretty verbose

Reply:
found an alternative whose output is easier to read